__fetch_polyfill not found in @sveltejs/kit/dist/install-fetch.js

```
Build failed with 1 error:
.svelte-kit/appengine-tmp/entry.js:3:8: error: No matching export in "node_modules/@sveltejs/kit/dist/install-fetch.js" for import "__fetch_polyfill"
```

Looks like the `__fetch_polyfill` is now available as `installFetch` as noted here: https://github.com/sveltejs/kit/pull/4111

--- a/files/entry.js
+++ b/files/entry.js
@@ -1,6 +1,6 @@
 import path from 'node:path';
 // eslint-disable-next-line camelcase
-import {__fetch_polyfill} from '@sveltejs/kit/install-fetch';
+import {installFetch} from '@sveltejs/kit/install-fetch';
 import {getRequest, setResponse} from '@sveltejs/kit/node';
 import compression from 'compression';
 import {manifest} from 'MANIFEST';
@@ -8,7 +8,7 @@ import polka from 'polka';
 import sirv from 'sirv';
 import {Server} from 'SERVER';
 
-__fetch_polyfill();
+installFetch();
 
 const app = new Server(manifest);
 

--- a/files/entry.js
+++ b/files/entry.js
@@ -1,5 +1,4 @@
 import path from 'node:path';
-// eslint-disable-next-line camelcase
 import {installFetch} from '@sveltejs/kit/install-fetch';
 import {getRequest, setResponse} from '@sveltejs/kit/node';
 import compression from 'compression';

--- a/index.js
+++ b/index.js
@@ -26,9 +26,7 @@ export default function entrypoint() {
       const relativePath = posix.relative(temporary, builder.getServerDirectory());
 
       builder.log.minor('Prerendering static pages');
-      const prerenderedPaths = await builder.prerender({
-        dest: `${dir}/storage`,
-      });
+      builder.writePrerendered(`${dir}/storage`);
 
       // Copy server handler
       builder.copy(files, temporary, {replace: {
@@ -53,14 +51,14 @@ export default function entrypoint() {
 
       writeFileSync(`${dir}/package.json`, JSON.stringify({type: 'commonjs'}));
 
-      const prerenderedPages = Array.from(prerenderedPaths.pages, ([src, page]) => ({
+      const prerenderedPages = Array.from(builder.prerendered.pages, ([src, page]) => ({
         url: src + '/?$',
         // eslint-disable-next-line camelcase
         static_files: 'storage/' + page.file,
         upload: 'storage/' + page.file,
       }));
 
-      const prerenderedRedirects = Array.from(prerenderedPaths.redirects, ([src, _]) => ({
+      const prerenderedRedirects = Array.from(builder.prerendered.redirects, ([src, _]) => ({
         url: src,
         secure: 'always',
         script: 'auto',
@@ -78,9 +76,9 @@ export default function entrypoint() {
         ...prerenderedPages,
         ...prerenderedRedirects,
         {
-          url: `/${builder.appDir}/`,
+          url: `/${builder.config.kit.appDir}/`,
           // eslint-disable-next-line camelcase
-          static_dir: `storage/${builder.appDir}`,
+          static_dir: `storage/${builder.config.kit.appDir}`,
           expiration: '30d 0h',
         },
         {

--- a/tests/expected_app.yaml
+++ b/tests/expected_app.yaml
@@ -11,7 +11,7 @@ handlers:
   - url: /about/?$
     static_files: storage/about.html
     upload: storage/about.html
-  - url: /_app/.+
+  - url: /_app/
     static_dir: storage/_app
     expiration: 30d 0h
   - url: /.*

--- a/tests/test.sh
+++ b/tests/test.sh
@@ -16,8 +16,8 @@ pushd $TEST_DIR
 
 set -e
 
-npm i
 npm i "${SCRIPT_PATH}/../"
+npm i
 
 # These are peer dependencies that need manual install since we install from folder instead of from npm registry
 npm install polka@1.0.0-next.22 compression@^1.7.4 sirv@^2.0.2

